### PR TITLE
Feature : Enforce user to accept privacy policies

### PR DESCRIPTION
**Pull request by icepick4**
Closes: https://github.com/origranot/reduced.to/issues/529

Hey,

I added a checkbox for the privacy policies, now the user must check it to register.

Squash my commits I made some shitty commits.

### CODE REVIEW

**Comment by icepick4** [comment]
Fixes issue #529

**Comment by icepick4** [comment]
image
image

File: apps/frontend/src/routes/register/index.tsx [hunk 1]
```diff
@@ -159,6 +166,21 @@ export default component$(() => {
...
                   </span>
                 </label>
+                <div class="flex items-center relative">
+                    <input
+                        name="policies"
+                        type="checkbox"
+                        class="checkbox checkbox-accent"
+                    />
+                     <span class="ml-2 text-xs text-left">
+                        I agree with the <a href="/privacy-policy" class="text-accent">Privacy Policy</a>.
```
Comment (origranot) [inline apps/frontend/src/routes/register/index.tsx:176]: Please change the class into `text-primary`

File: apps/frontend/src/routes/register/index.tsx [hunk 2]
```diff
@@ -159,6 +166,21 @@ export default component$(() => {
...
+                    </span>
+                </div>
+                <label class="label">
+                    {action.value?.fieldErrors?.policies &&<span class="label-text text-error text-left">
+                    {action.value?.fieldErrors?.policies}
```
Comment (origranot) [inline apps/frontend/src/routes/register/index.tsx:181]: You don't need to use the optional chaining here.
```suggestion
                    {action.value.fieldErrors.policies}
```

**Review by origranot — CHANGES_REQUESTED** [review]
@icepick4 Hey,

I really liked it, nice job!!
I left a few comments, please review them 💯 

Let's merge it 🚀

File: apps/frontend/src/routes/register/index.tsx (hunk 1 above)
Comment (origranot) [inline apps/frontend/src/routes/register/index.tsx:176]: Maybe it worth open the link in a new tab?

**Pull request merged**